ONI-DenseLogic/RemapperSideScreen.cs

```diff
@@ -135,5 +164,75 @@ private sealed class BitOption : IListableOption {
 				return name;
 			}
 		}
+
+		private sealed class RemapperRow {
+			private RemapperSideScreen parent;
+
+			/// <summary>
+			/// The panel containing this row
+			/// </summary>
+			public PPanel Row { get; }
+
+			public RemapperRow(int index, RemapperSideScreen parent) {
+				// TODO This could probably be done as a relative layout eventually, but
+				// until PRelativePanel is added this is the best we can do
+				this.parent = parent;
+				Row = new PPanel("RemapperRow") {
+					Margin = new RectOffset(4, 4, 4, 4),
+					FlexSize = new Vector2(1.0f, 0.0f),
+				};
+				var RowBackground = new PPanel("RemapperRowBackground") {
+					FlexSize = new Vector2(1.0f, 0.0f)
+				};
+				RowBackground.OnRealize += gameObject => {
+					var img = gameObject.AddComponent<KImage>();
+					img.sprite = PUITuning.Images.GetSpriteByName("BitSelectorSideScreenRow");
```

Comment:
If you do not need the KImage features like color style settings, background sprite and color can be set in the PPanel constructor (there is a TODO item for more panel features in general across all panel types in PLib)